src/main/features/claim/views/confirmation.njk

```diff
@@ -20,52 +20,34 @@
       </div>
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
-      <p>{{ t('%s needs to respond to your claim by %s',
+      <p>{{ t('%s has until 4pm on %s to respond to your claim. They can ask for an extra 14 days if they need it.
+              We’ll email you when they respond.',
         {
           postProcess: 'sprintf',
           sprintf: [ defendantName, responseDeadline | date ]
         }
         ) | safe }}
       </p>
-      <p>{{ t('We’ll send you an update by email if they do respond, and we’ll explain what to do if they don’t.') }}</p>
+      <p>{{ t('You can request a default County Court Judgment (CCJ) against them if they don’t respond.') }}</p>
 
-      <p>{{ t('You can <a href="%s">view your claim</a> at any time.',
-        {
-          postProcess: 'sprintf',
-          sprintf: ['/dashboard']
-        }) | safe }}</p>
-
-      <h2 class="heading-medium">{{ t('What you can do') }}</h2>
+      <h2 class="heading-medium">{{ t('If the defendant pays you') }}</h2>
       <p>
-        {{ t('You can still contact %s directly while your claim progresses. ' +
-        'Many people are able to settle their claim by discussing what’s gone wrong. ' +
-        'If you can make an agreement you won’t need to go to a hearing.',
+        {{ t('You need to email moneyclaims@hmcts.gsi.gov.uk to let us know. Include the date you were paid.',
```

Comment:
uhh  this should come from configuration (its already there)

Reply:
done